src/utils/Git.ts

```diff
@@ -127,16 +126,22 @@ export async function branches({
         ['"symbolic-ref"', '"--short"', '"-q"', '"HEAD"'],
         directory)
         .then(branchName => branchName.trim());
+    const branchFilter = [
+        "s/^refs\\/heads\\///g",
+        remotes ? "s/^refs\\/remotes\\/[^\\?%\\*\\:\\|\"\\<\\>\\\\]*\\///g" : "",
+        tags ? "s/^refs\\/tags\\///g" : ""].join(";");
```

Comment:
This seems really complicated. Can we do two calls, one for heads and tags with `--format='%(HEAD)%(refname:strip=2)'`, and one for remotes with `strip=3`, as it's done in bash completion? https://github.com/git/git/blob/master/contrib/completion/git-completion.bash